src/components/navigation/Logout.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright 2018-2021 Cargill Incorporated
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import Icon from '@material-ui/core/Icon';
+import { get } from '../../request';
+
+export const Logout = () => {
+  const onLogout = async () => {
+    const { splinterURL } = window.$CANOPY.getSharedConfig().canopyConfig;
+    const { token } = JSON.parse(window.sessionStorage.getItem('CANOPY_USER'));
```

Comment:
This session storage may have expired, so you should check if the value exists before proceeding.  If not, just perform the redirect. An exception would occur if that's the case currently.

Reply:
Added a check for `CANOPY_USER` in session storage before proceeding with logout